Review Assignment/src/assets/scripts/sidebar.js

```diff
@@ -0,0 +1,136 @@
+      for (var i in  update.people) {
+        let currIndex = update.people[i];
+        if (clickedElement == currIndex.sno) {
+          followings.people.push({'name': currIndex.name, 'flag': "true", 'job': currIndex.job, 'sno': currIndex.sno, 'image': currIndex.image, 'follow': currIndex.follow});
+          newFriend = {list: [{'name': currIndex.name, 'flag': "true", 'job': currIndex.job, 'sno': currIndex.sno, 'image': currIndex.image, 'follow': currIndex.follow}]};
+        } else {
+          followings.people.push({'name': currIndex.name, 'flag': currIndex.flag, 'job': currIndex.job, 'sno': currIndex.sno, 'image': currIndex.image, 'follow': currIndex.follow});
+        }
+        localStorage.setItem('followings', JSON.stringify(followings));
+      }
+      if (totalElements == counter) {
+        $('.suggestion-container').addClass(hidden);
+        localStorage.setItem('suggetionsDisplay', 'false');
+      }
+      updateFollowings();
+    });
+  }
+
+  function defaultDataForSidebarFriends() {
+    const dataUrl = '../assets/data/friends.json';
+    const templateUrl = '../assets/templates/sidebar-friends.mustache';
+    $.get({url: dataUrl, type: type, success: function(data) {
+      $.get({url: templateUrl, type: type, success: function(template) {
+        var showFriends = Mustache.render(template, data)
+        $('.friends-body').html(showFriends);
+        $('#tile-5').nextAll().addClass( "hidden");
+      }});
+    }});
+  }
+
+  //See more buttton functionality
+  function displaySeeMore() {
+    defaultDataForSidebarFriends();
+    var seeMoreBtn = $('.see-more-btn');
+    seeMoreBtn.on('click', function(event) {
+      if (seeMoreBtn.text() == 'See all') {
+        const dataUrl = '../assets/data/friends.json';
+        const templateUrl = '../assets/templates/sidebar-friends.mustache';
+        $.get({url: dataUrl, type: type, success: function(data) {
+          $.get({url: templateUrl, type: type, success: function(template) {
+            var showFriends = Mustache.render(template, data)
+            $('.friends-body').html(showFriends);
+          }});
+        }});
+        seeMoreBtn.text('Hide');
+      } else {
+        defaultDataForSidebarFriends();
+        seeMoreBtn.text('See all');
+      }
+    });
+  }
+
+  function suggetionsDisplay() {
+    var show = localStorage.getItem('suggetionsDisplay');
+    if (show == 'false') {
+      $('.suggestion-container').addClass(hidden);
+    }
+  }
+
+  function displaySidebar() {
+    const dataUrl = '../assets/data/friends.json';
+    const templateUrl = '../assets/templates/sidebar.mustache';
+
+    $.get({url: dataUrl, type: type, success: function(suggetions) {
+      $.get({url: templateUrl, success: function(sidebar) {
+        var sideBar = Mustache.render(sidebar, suggetions)
+        $('#side-bar').html(sideBar);
+        displaySeeMore();
+        followButtonClick();
+        displaySuggestion();
+        $('.data-true').addClass(hidden);
```

Comment:
`.data-true` class name should be contextual or generalise. It sounds nothing now.